CHANGELOG.md

```diff
@@ -8,6 +8,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 ## [Unreleased]
 ### Added
 
+- Provide pin definitions for A0 to A7.
```

Comment:
Most platforms have such definitions. Maybe this can be controlled using a `NUM_ANALOG_PINS` (which would be good to define in any case, along with `NUM_DIGITAL_PINS`, since those are at least defined for AVR-based boards: https://github.com/rlcamp/ArduinoCore-avr/blob/4c9f899f8eef12989903e474beea3a71a90f511c/variants/standard/pins_arduino.h#L28-L29)